openlibrary/macros/StarRatingsStats.html

```diff
@@ -6,7 +6,7 @@
 <ul class="readers-stats" itemprop="aggregateRating" itemscope itemtype="https://schema.org/AggregateRating">
   <li class="avg-ratings">
     $if rating_stats.get('avg_rating'):
-      $:('<span class="readers-stats__star">★</span>' * int((int(2 * float(rating_stats.get('avg_rating')))) - (int(rating_stats.get('avg_rating'))))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
+      $:('<span class="readers-stats__star">★</span>' * round(float(rating_stats.get('avg_rating')))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
```

Comment:
Sorry.  Revert this.  Do it your way.  It will take time for the upgrade to web.py to be reviewed and landed and then incorporated into https://github.com/internetarchive/openlibrary/blob/master/requirements_common.txt#L28-L29